api/apps/api/src/modules/proxy/proxy.service.ts

```diff
@@ -1,67 +1,27 @@
-import { HttpStatus, Injectable, Logger } from '@nestjs/common';
+import { Injectable, Logger } from '@nestjs/common';
 
 import { AppConfig } from '@marxan-api/utils/config.utils';
 import { Request, Response } from 'express';
-/**
- * @debt i think this is not properly imported, but it was the only way for it to work
- */
-import * as Server from 'http-proxy';
+import { get } from 'http';
```

Comment:
alternatively we could use HttpService with `responseType: 'stream'` like @kgajowy did in `AssetFetcher`